save-backend/src/main/kotlin/com/saveourtool/save/backend/storage/AvatarStorage.kt

```diff
@@ -30,15 +33,28 @@ class AvatarStorage(configProperties: ConfigProperties) :
         pathToContent.name
     )
 
+    /**
+     * @param key
+     * @param content
+     * @return `Mono` with file size
+     */
+    fun upsert(key: AvatarKey, content: Flux<ByteBuffer>): Mono<Long> {
+        list()
+            .filter { it.imageName == key.folderUserName }
```

Comment:
why did you compare `imageName` with `folderUserName`?

Reply:
Yes. Thank you. fixed